Gems/AtomLyIntegration/CommonFeatures/Code/Source/DiffuseGlobalIllumination/DiffuseProbeGridComponentController.cpp

```diff
@@ -34,12 +34,13 @@ namespace AZ
             if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
             {
                 serializeContext->Class<DiffuseProbeGridComponentConfig>()
-                    ->Version(1)
+                    ->Version(2) // ATOM-17127
```

Comment:
nit: since this is Jira ticket, I'd suggest adding short 4-5 words about the change

Reply:
Good catch, I'll change it to a description.